packages/preview/exzellenz-tum-thesis/0.1.0/disclaimer.typ

```diff
@@ -0,0 +1,31 @@
+#let disclaimer(
+  title: "",
+  degree: "",
+  author: "",
+  submissionDate: none,
+) = {
+
+  // --- Disclaimer ---
+  pagebreak()
+  
+  v(1fr)
+  text("Disclaimer", weight: 600, size: 1.4em)
+
+  v(1.5em)
+  
+  text("I confirm that this " + degree + "’s thesis is my own work and I have documented all sources and material used.", size: 1.1em)
+
+  v(25mm)
+  grid(
+      columns: 2,
+      gutter: 1fr,
+      //"Munich, " + submissionDate,
+      overline[#sym.wj #sym.space #sym.space #sym.space #sym.space (Date) #sym.space #sym.space #sym.space #sym.space #sym.wj],
+      overline[#sym.wj #sym.space #sym.space #sym.space #sym.space #sym.space (#author) #sym.space #sym.space #sym.space #sym.space #sym.space #sym.wj]
```

Comment:
When the space takes overhand, you can generate them with a loop. Alternatively, you could also use `h`. But that's not blocking for acceptance.